src/GameKit/GameKit.cs

```diff
@@ -291,11 +291,43 @@ public enum GKInviteRecipientResponse : long {
 		NoAnswer = 5,
 	}
 
-	[Mac (10,13,4), TV (11,3), iOS (11,3)]
+	[iOS (11,3)][Obsoleted (PlatformName.iOS, 14,0, message: "Do not use; this API was removed.")]
+	[Mac (10,13,4)][Obsoleted (PlatformName.MacOSX, 10,16, message: "Do not use; this API was removed.")]
+	[TV (11,3)][Obsoleted (PlatformName.TvOS, 14,0, message: "Do not use; this API was removed.")]
+	[Obsolete]
```

Comment:
we normally use `[Obsolete]`, with a message, only for our binding _mistakes_
and remove them under `#if !XAMCORE_4_0`
so customers can easily spot what they should avoid for whenever (and if) we do breaking changes

Reply:
Updated these to `Deprecated` w/ a message to let users know that these APIs no longer exist since Apple removed them. Lmk if there's a better way to do this!